flow/core/params.py

```diff
@@ -522,7 +549,7 @@ def add(self,
             probability=None,
             number=None,
             **kwargs):
-        """Specifies a new inflow for a given type of vehicles and edge.
+        r"""Specify a new inflow for a given type of vehicles and edge.
```

Comment:
Don't think this r is intended?

Reply:
docstring standards is complaining about this